src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -15352,9 +15352,6 @@ private bool example2()
 }
 ";
             CreateStandardCompilation(text, options: TestOptions.UnsafeReleaseDll).VerifyDiagnostics(
-                // (13,17): error CS1666: You cannot use fixed size buffers contained in unfixed expressions. Try using the fixed statement.
-                //         return (field.buffer[0] == 0);   // CS1666 error
-                Diagnostic(ErrorCode.ERR_FixedBufferNotFixed, "field.buffer")
```

Comment:
>Diagnostic(ErrorCode.ERR_FixedBufferNotFixed, "field.buffer") [](start = 16, length = 61)

Since this is no longer an error, it would be good to compile and verify the code, run it and observe expected behavior. If we have a test like this elsewhere, consider removing this unit-test to avoid duplication.